citrus-remote-maven-plugin/pom.xml

```diff
@@ -1,11 +1,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
     <artifactId>citrus-remote</artifactId>
-    <groupId>com.consol.citrus</groupId>
-    <version>3.5.0-SNAPSHOT</version>
+    <groupId>org.citrusframework</groupId>
+    <version>4.3.1-SNAPSHOT</version>
```

Comment:
There is no `4.3.1-SNAPSHOT`. The snapshot versions always use the next major version. This means you need to use `4.4.0-SNAPSHOT` here

Reply:
I see, I just updated it 😃

Reply:
Thank you! I think we are ready to merge then 🥳